src/views/projectPageWebsite.hbs

```diff
@@ -12,14 +12,14 @@
   </h1>
   <ul class="information-ui">
     <li> {{res.projectResult.0.cohort_name}} </li>
```

Comment:
Do not send the entire response to handlebars and deal with it later, send only the cohort name as `cohortName` and whatever you need out of the result.